### ISSUE TITLE
DOM order doesn't match visual order for min/max/restore buttons

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5997]
### System details

    RStudio Edition : Desktop or Server 
    RStudio Version : 1.3.x (see note about related PR that exposes this)
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
Note, this only becomes evident once this PR is accepted: https://github.com/rstudio/rstudio/pull/5996

- tab (or navigate via screen reader cursor) to/from the little buttons in the 4 quadrants for min/max/restore of panes, e.g.:

<img width="56" alt="screenshot showing representative example of the min/max/restore buttons" src="https://user-images.githubusercontent.com/10569626/72181439-337e8380-339e-11ea-9b9a-1de32db50ed4.png">

### Describe the problem in detail
- the DOM order doesn't match the visual order, so behavior isn't what a keyboard user expects
- a WCAG violation: https://www.w3.org/TR/WCAG21/#focus-order

### Describe the behavior you expected
- generally, tabbing should proceed left-to-right / top-down visual layout
- caveat, more of a compliance issue than a significant problem: unsighted users aren't likely to care much about these controls, and keyboard users will be annoyed but able to perform the tasks (can also use View menu to accomplish such resizing)
